server/migrations/20200411191438-add-reviewed-field-and-indexes.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return Promise.all([
+        queryInterface.addColumn("StoreUpdates", "reviewed", {
+          type: Sequelize.BOOLEAN,
+          defaultValue: false,
+          allowNull: false,
+        }),
+        queryInterface.addIndex(
+          'StoreUpdates',
+          ['deleted']
+        ),
+        queryInterface.addIndex(
+          'StoreUpdates',
+          ['reviewed']
+        ),
+        queryInterface.addIndex(
```

Comment:
why do we have index on flag?

Reply:
`deleted` i get, since we are querying on it. not clear why indexes on flag and reviewed are required

Reply:
We'll be querying by what's flagged and what's been reviewed when getting the dump of new reviews to moderate.

Reply:
since these queries aren't time bound (unlike API requests to the frontend which need to have fast response times), i don't think we should add these indexes without verifying we need them, since they will slow submissions (another API that needs to be fast). with 1000 rows, i think the script should be able to manage - please correct me if i'm wrong

Reply:
You're right, those queries aren't time-bound. However, if they take a while, we could be tying up DB resources. I agree that with fewer rows it should be ok, and we can revisit if needed later. It's a lot harder to index with more data though, but we could schedule in a DB update time. Happy to remove them for now, was thinking of having to scale this later.

Reply:
I was thinking of running read queries off a follower database. The results can be written to production with their id fields - which is indexed.

Reply:
Fair enough, will take these indexes out.